haystack/pipelines/base.py

```diff
@@ -1456,9 +1456,11 @@ def _reorder_columns_in_eval_result(self, eval_result: EvaluationResult) -> Eval
             "gold_id_and_context_and_answer_match",  # doc-specific
             "context_and_answer_match",  # doc-specific
             "rank",  # generic
-            "document_id",  # generic
+            "document_id",  # document-specific
+            "document_ids",  # answer-specific
             "gold_document_ids",  # generic
-            "custom_document_id",  # generic
+            "custom_document_id",  # document-specific
+            "custom_document_ids",  # answer-specific
```

Comment:
Previously we had only `document_id` and `custom_document_id` in both eval CSVs: the one for answers and the one for documents.
As answers can have multiple document_ids (instead of documents which still have only one), we use different columns to reflect the different semantics (and types).